src/components/controls/base/BaseSearchBar.vue

```diff
@@ -0,0 +1,30 @@
+<template>
+  <div>
+    <input
+      :value="value"
+      @input="$emit('input', $event.target.value)"
+      id="baseSearch"
+      type="text"
+      :placeholder="placeholderText"
+    />
+    <img :src="iconSrc" />
+  </div>
+</template>
+
+<script>
+export default {
+  props: {
+    value: {
+      type: String,
+      default: "",
+    },
+    placeholderText: {
+      type: String,
+      default: "",
+    },
+    iconSrc: {
+      type: String,
+    },
```

Comment:
Let's give a default here, for now it can be an empty string, but eventually can be something related to search.